conductr_cli/sandbox_init.py

```diff
@@ -1,62 +1,73 @@
-from conductr_cli import docker_machine, terminal
+from conductr_cli import docker, docker_machine, terminal
+from conductr_cli.docker import DockerVmType
 import logging
 from subprocess import CalledProcessError
 
 
 def init(args):
     """`sandbox init` command"""
 
-    log = logging.getLogger(__name__)
-    vm_name = docker_machine.vm_name()
-    if is_docker_machine_installed():
```

Comment:
I *think* docker-machine can be installed on Linux also... so perhaps we wish to keep this logic around.